src/libsync/propagateremotedelete.cpp

```diff
@@ -156,6 +156,7 @@ void PropagateRemoteDelete::slotDeleteJobFinished()
     }
 
     propagator()->_journal->deleteFileRecord(_item->_originalFile, _item->isDirectory());
+	propagator()->_journal->deleteSyncMode(_item->_originalFile);
```

Comment:
Please fix indentation

I also wonder if deleteFileRecord shouldn't do this by itself.